packages/envied_generator/lib/src/generator.dart

```diff
@@ -54,7 +53,7 @@ final class EnviedGenerator extends GeneratorForAnnotation<Envied> {
         '// ignore_for_file: type=lint\n'
         '// generated_from: $generatedFrom';
 
-    return DartFormatter().format('$ignore\n$generatedClassesAltogether');
+    return '$ignore\n$generatedClassesAltogether';
```

Comment:
The tests pass with no changes after removing the call to `format`, so I don't think we have a need for it.

Reply:
Not sure I like that. Could you please change it to
```dart
return DartFormatter(languageVersion: Version(3, 0, 0))
       .format('$ignore\n$generatedClassesAltogether');
```

Reply:
OK!